### PR TITLE
Fixes#3 check python version

### PR DESCRIPTION
**Pull request by Jasminewwwu**
some warning messages will show up if wrong python version is detected

### CODE REVIEW

File: pyta.py [hunk 1]
```diff
@@ -26,6 +27,9 @@ def check(module_name):
     The name of the module should be passed in as a string,
     without a file extension (.py).
     """
+    if sys.version_info<(3,3,0):
```
Comment (david-yz-liu) [inline pyta.py]: Please put the necessary whitespace

File: pyta.py [hunk 2]
```diff
@@ -26,6 +27,9 @@ def check(module_name):
...
+        sys.stderr.write("You need python 3.3 or later to run this script\n")
```
Comment (david-yz-liu) [inline pyta.py]: Please use single-quotes; also, only allow Python version 3.4+. And also, please capitalize "Python."

File: pyta.py [hunk 3]
```diff
@@ -14,11 +14,17 @@
...
 import os
+import sys
 from urllib.request import pathname2url
 
 # Local version of website; will be updated later.
 HELP_URL = 'file:' + pathname2url(os.path.abspath('website/index.html'))
 
+# check the python version
+if sys.version_info > (3,4,0):
+    sys.stderr.write('You need Python 3.4 or later to run this script\n')
```
Comment (nigef) [inline pyta.py]: maybe say what this script is

File: pyta.py [hunk 4]
```diff
@@ -14,11 +14,17 @@
...
 import webbrowser
 import os
+import sys
 from urllib.request import pathname2url
 
 # Local version of website; will be updated later.
 HELP_URL = 'file:' + pathname2url(os.path.abspath('website/index.html'))
 
+# check the python version
+if sys.version_info > (3,4,0):
```
Comment (david-yz-liu) [inline pyta.py]: @Jasminewwwu put spaces after commas as well.

File: pyta.py (hunk 3 above)
Comment (david-yz-liu) [inline pyta.py]: Don't use stderr (print is fine), and the version check is not actually for running PyTA, but for the course itself.

Comment (Jasminewwwu) [inline pyta.py]: Then where should I put these lines?

Comment (david-yz-liu) [inline pyta.py]: As I said, use `print`

File: pyta.py [hunk 5]
```diff
@@ -14,11 +14,17 @@
...
+import sys
 from urllib.request import pathname2url
 
 # Local version of website; will be updated later.
 HELP_URL = 'file:' + pathname2url(os.path.abspath('website/index.html'))
 
+# check the python version
+if sys.version_info < (3, 4, 0):
+    print('You need Python 3.4 or later to run this script\n')
+    exit(1)
```
Comment (david-yz-liu) [inline pyta.py]: @Jasminewwwu I realized that there shouldn't actually be an `exit` here. Just the warning message is enough.

**Comment by david-yz-liu** [comment]
Thanks, @Jasminewwwu!

**Pull request merged**